crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -471,14 +468,12 @@ impl GetBatchData for MeshPipeline {
 
     fn get_batch_data(
         (mesh_instances, lightmaps): &SystemParamItem<Self::Param>,
-        entity: &QueryItem<Self::Data>,
-    ) -> (Self::BufferData, Option<Self::CompareData>) {
-        let mesh_instance = mesh_instances
-            .get(entity)
-            .expect("Failed to find render mesh instance");
+        entity: &Entity,
```

Comment:
Entity is both Copy and two u32s. It's reasonable to pass by value instead of by reference.